### ISSUE TITLE
iOS: "Module RNUportHDSigner requires main queue setup"

### ISSUE THREAD

**Issue by simonas-notcat** [uport-project/react-native-uport-signer#10]
New RN app using `"react-native": "0.59.5"`.

Getting this warning:

```
Module RNUportHDSigner requires main queue setup since it overrides `constantsToExport` but doesn't implement `requiresMainQueueSetup`. In a future release React Native will default to initializing all native modules on a background thread unless explicitly opted-out of.
```

![Simulator Screen Shot - iPhone X - 2019-04-23 at 17 43 17](https://user-images.githubusercontent.com/16773277/56590562-f98f0680-65ef-11e9-9954-941da6f4a1c5.png)

**Comment by simonas-notcat** [comment]
@aldigjo you probably only need to add:

```
+ (BOOL)requiresMainQueueSetup
{
    return NO;
}
```

**Comment by mirceanis** [comment]
fixed in #12
